src/_kw

```diff
@@ -405,7 +406,7 @@ _kw_man()
   _values 'kw commands' \
     'backup' 'build' 'codestyle' 'config' 'debug' 'deploy' 'device' 'diff' 'drm' 'env' \
     'explore' 'init' 'mail' 'maintainers' 'pomodoro' 'remote' 'report' 'self-update' 'ssh' \
-    'patch-hub' 
+    'patch-hub'
```

Comment:
Is this change part of this PR, or is this unrelated?

Reply:
No, this shouldn't be here.